### ISSUE TITLE
Invalid nested section warning

### ISSUE THREAD

**Issue by rafaelrenanpacheco** [asciidoctor/asciidoctor-intellij-plugin#362]
AsciiDoc Plugin Version: 0.30.7

IDE Name and Version: IntelliJ IDEA 2019.2.3 (Community Edition)

Hello! I'm trying to create a nested menu and for some reason the plugin is warning about it:

![image](https://user-images.githubusercontent.com/12160864/67813046-5841ab00-fa7f-11e9-8e26-b32312fc82c1.png)

When I generate the docs using Spring REST Docs the menu is generated just fine. The AsciiDoc Plugin preview also renders the menu and its submenus perfectly. So it looks like a linter problem, and not a syntax problem.

Here is the sample that triggers that warning:

```
:doctype: book
:icons: font
:source-highlighter: highlightjs
:toc: left
:toclevels: 4
:sectlinks:

= Formulários
Endpoints de formulários

[gerar-tokens]
== Gerar tokens
Gera um token para o formulário.

[gerar-tokens-request]
=== Request
Exemplo de requisição para gerar um token para um formulário:

include::{snippets}/post/forms/generate_token/http-request.adoc[]
include::{snippets}/post/forms/generate_token/path-parameters.adoc[]

[gerar-tokens-response]
=== Response
O retorno será um JSON contendo o token (UUID v4) que foi associado ao formulário:

include::{snippets}/post/forms/generate_token/response-body.adoc[]
```

**Comment by rafaelrenanpacheco** [comment]
Actually this error is showing on console as well:

```
[INFO] asciidoctor: ERROR: FormController.adoc: line 16: gerar-tokens sections do not support nested sections
[INFO] asciidoctor: ERROR: FormController.adoc: line 23: gerar-tokens sections do not support nested sections
```

I'll try to figure it out.

Thanks.

**Comment by ahus1** [comment]
I see you added `[gerar-tokens-request]` to each section. I assume these should be the IDs you want to reference the sections with later using <<gerar-tokens-request>>, so they should have double brackets like `[[gerar-tokens-request]]`.

Single brackets are for section types like `[appendix]`.

[There used to be an example for spring-rest-docs that caused some confusion. It's now fixed.](https://github.com/spring-projects/spring-restdocs/commit/4a084b52d06f149c62317985e302eeaed7487c35) 

If you find other examples that confuse these two, please notify the their owners.

**Comment by rafaelrenanpacheco** [comment]
Hello @ahus1!

After changing to double brackets the warning went away, thanks for the help!

My doc was based on the v2.0.3 of Spring REST Docs, and from that link they fixed it on v2.0.4.

Thanks!
